hw/ip/clkmgr/dv/clkmgr_sim_cfg.hjson

```diff
@@ -80,6 +81,31 @@
       name: clkmgr_clk_status
       uvm_test_seq: clkmgr_clk_status_vseq
     }
+    {
+      name: clkmgr_idle_intersig_mubi
+      uvm_test_seq: clkmgr_trans_vseq
+      run_opts: ["+clkmgr_mubi_mode=1"]
```

Comment:
it's better to use enum name and use this macro to obtain it
https://github.com/lowRISC/opentitan/blob/a609d5560844803b798391f76c5f87a11219f16c/hw/dv/sv/dv_utils/dv_macros.svh#L426

Reply:
good to know we have a macro to handle enum by name.
fixed.